TestStudies.fs

```diff
@@ -54,92 +52,337 @@ method unlock() {
 
 /// The correct parsing of the ticket lock's lock method.
 let ticketLockLockMethodAST =
-    { Signature = {Name = "lock"; Params = []}
-      Body =
-          { Pre = Unmarked Unit
-            Contents =
-                [ { Command =
-                        Command.Prim
-                            { PreAssigns = []
-                              Atomics = [ Fetch(LVIdent "t",
-                                                LV(LVIdent "ticket"),
-                                                Increment) ]
-                              PostAssigns = [] }
-                    Post = Unmarked <|
-                           View.Func {Name = "holdTick"
-                                      Params = [ LV(LVIdent "t") ]} }
-                  { Command =
-                        DoWhile
-                            ({ Pre = Unmarked <|
-                                     View.Func {Name = "holdTick"
-                                                Params = [ LV(LVIdent "t") ]}
-                               Contents =
-                                   [ { Command =
-                                           Command.Prim
-                                               { PreAssigns = []
-                                                 Atomics = [ Fetch(LVIdent "s",
-                                                                   LV(LVIdent "serving"),
-                                                                   Direct) ]
-                                                 PostAssigns = [] }
-                                       Post =
-                                           Unmarked <|
-                                           View.If
-                                               (Bop(Eq, LV(LVIdent "s"), LV(LVIdent "t")), View.Func {Name = "holdLock"; Params = []},
-                                                View.Func {Name = "holdTick"; Params = [ LV(LVIdent "t") ]}) } ] },
-                             Bop(Neq, LV(LVIdent "s"), LV(LVIdent "t")))
-                    Post = Unmarked <|
-                           View.Func { Name = "holdLock"; Params = [] } } ] } }
+    {Signature = {Name = "lock";
+                  Params = [];};
+           Body =
+            {Pre = Unmarked Unit;
+             Contents =
+              [{Command =
+                 {Position = {StreamName = "Examples/Pass/ticketLock.cvf";
```

Comment:
Might be worth making a function to build these, there's a lot of repetition here